source/core/src/main/com/csse3200/game/ui/UIComponent.java

```diff
@@ -13,7 +13,7 @@
 public abstract class UIComponent extends RenderComponent implements Renderable {
   private static final int UI_LAYER = 2;
   protected static final Skin skin =
-      new Skin(Gdx.files.internal("flat-earth/skin/flat-earth-ui.json"));
+          new Skin(Gdx.files.internal("flat-earth/skin/flat-earth-ui.json"));
```

Comment:
Seems like these sorts of changes are due to an auto-formatter changing spaces/tabs, might be best to revert these files that don't have any effect on this pull request, and run an auto-formatter in a separate branch/issue.